spec/services/github_org_membership_service_spec.rb

```diff
@@ -146,33 +146,87 @@ def build(*_args)
-        service.import_default_team_members(organization)
+        service.import_default_team_members(organization, false)
       end
 
       it 'resets all default team memberships to false' do
-        expect(OrganizationMembership.find_by(github_user_id: user.id).is_member_of_default_team)
+        expect(OrganizationMembership.find_by(github_user_id: user.id, organization_id: organization.id)
+          .is_member_of_default_team)
           .to be(false)
-        expect(OrganizationMembership.find_by(github_user_id: user2.id).is_member_of_default_team)
+        expect(OrganizationMembership.find_by(github_user_id: user2.id, organization_id: organization.id)
+          .is_member_of_default_team)
           .to be(false)
       end
     end
 
-    context 'when not empty team is selected as default' do
+    context 'when not empty github team is selected as default' do
       before do
         allow(client).to receive(:team_members)
           .with(organization.default_team_id).and_return([gh_user])
-        service.import_default_team_members(organization)
+        service.import_default_team_members(organization, false)
+      end
+
+      it 'asigns default team membership to correct users' do
+        expect(OrganizationMembership.find_by(github_user_id: user.id, organization_id: organization.id).is_member_of_default_team)
+          .to be(true)
+        expect(OrganizationMembership.find_by(github_user_id: user2.id, organization_id: organization.id).is_member_of_default_team)
+          .to be(false)
+      end
+    end
+
+    context 'when empty froggo team is selected as default' do
+      before do
+        service.import_default_team_members(organization2, true)
+      end
+
+      it 'resets all default team memberships to false' do
+        expect(OrganizationMembership.find_by(github_user_id: user.id, organization_id: organization2.id)
+          .is_member_of_default_team)
+          .to be(false)
+        expect(OrganizationMembership.find_by(github_user_id: user2.id, organization_id: organization2.id)
+          .is_member_of_default_team)
+          .to be(false)
+      end
+    end
+
+    context 'when not empty froggo team is selected as default' do
+      let!(:membership) { create(:froggo_team_membership, github_user: user, froggo_team: froggo_team) }
+
+      before do
+        service.import_default_team_members(organization2, true)
       end
 
       it 'asigns default team membership to correct users' do
-        expect(OrganizationMembership.find_by(github_user_id: user.id).is_member_of_default_team)
+        expect(OrganizationMembership.find_by(github_user_id: user.id, organization_id: organization2.id)
+          .is_member_of_default_team)
           .to be(true)
-        expect(OrganizationMembership.find_by(github_user_id: user2.id).is_member_of_default_team)
+        expect(OrganizationMembership.find_by(github_user_id: user2.id, organization_id: organization2.id)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/LineLength: Line is too long. [106/100]